ports/servoshell/desktop/webview.rs

```diff
@@ -240,7 +245,10 @@ where
                         gamepad_event = Some(GamepadEvent::Disconnected(index));
                     },
                     EventType::ForceFeedbackEffectCompleted => {
-                        gamepad_event = Some(GamepadEvent::HapticEffectCompleted(index));
+                        self.haptic_effects[&event.id.into()]
```

Comment:
Should we not check for the presence of the effect? Is it possible for `stop_haptic_effect` to be called after `EventType::ForceFeedbackEffectCompleted` has already been enqueued but not handled yet(in which case the effect would have been removed already). Not sure what guarantees the `stop` calls give....